lib/main_common.pm

```diff
@@ -1827,7 +1827,7 @@ sub load_extra_tests_filesystem {
     loadtest "console/lsof";
     loadtest "console/autofs";
     loadtest 'console/lvm';
-    if (get_var("FILESYSTEM", "btrfs") eq "btrfs") {
+    if (check_var("FILESYSTEM", "btrfs") || (is_jeos && !is_openstack) || is_sle_micro || is_microos) {
```

Comment:
What's this change about?

Reply:
I wanted to fix this, but I have updated wrong line :facepalm:

```
Use of uninitialized value in pattern match (m//) at sle/lib/main_common.pm line 1853.
	main_common::load_extra_tests_filesystem() called at sle/lib/main_common.pm line 1800
	main_common::load_extra_tests() called at sle/products/sle/main.pm line 977
	require sle/products/sle/main.pm called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 394
	OpenQA::Isotovideo::Utils::try {...} () called at /usr/lib/perl5/vendor_perl/5.26.1/Try/Tiny.pm line 102
	eval {...} called at /usr/lib/perl5/vendor_perl/5.26.1/Try/Tiny.pm line 93
	Try::Tiny::try(CODE(0x5635b54811a8), Try::Tiny::Catch=REF(0x5635b4142ca8)) called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Utils.pm line 416
	OpenQA::Isotovideo::Utils::load_test_schedule() called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Runner.pm line 89
	OpenQA::Isotovideo::Runner::load_schedule(OpenQA::Isotovideo::Runner=HASH(0x5635b247c518)) called at /usr/lib/os-autoinst/OpenQA/Isotovideo/Runner.pm line 78
	OpenQA::Isotovideo::Runner::prepare(OpenQA::Isotovideo::Runner=HASH(0x5635b247c518)) called at /usr/bin/isotovideo line 171
	eval {...} called at /usr/bin/isotovideo line 161
```